pkg/features/isis/collector.go

```diff
@@ -77,6 +83,12 @@ func (c *isisCollector) Collect(client collector.Client, ch chan<- prometheus.Me
 		}
 	}
 
+	var i interfaces
```

Comment:
`i` is to be used for an int loop invariant. Would suggest `ifas` as short form of `interfaces`.

Reply:
Done